CHANGELOG.md

```diff
@@ -11,12 +11,21 @@ version directory, and then changes are introduced.
 
 
 
-## [v3.2.6] WIP
+## [v3.3.0] WIP
+
+### Changed
+- Updated hyperkube to version 1.10.1.
+
+### Removed
+
+
+## [v3.2.6]
```

Comment:
At the bottom of the file is a link reference. This version here is not linked and neither is the one above. The links help navigate quickly through the versions. Lets keep this in sync.